draft-ietf-tls-ctls.md

```diff
@@ -108,23 +118,10 @@ freedom. For example, if we only support one version of TLS, then it
 is not necessary to have version negotiation and the
 supported_versions extension can be omitted.
 
-Importantly, this process is performed only for the wire encoding but
-not for the handshake transcript.  The result is that the transcript for a
-specialized cTLS handshake is the same as the transcript for a TLS 1.3 handshake
-with the same features used.
-
-One way of thinking of this is as if specialization is a stateful compression
-layer between the handshake and the record layer:
-
-~~~~~
-+---------------+---------------+---------------+
-|   Handshake   |  Application  |     Alert     |
-+---------------+---------------+---------------+    +---------+
-|               cTLS Compression Layer          |<---| Profile |
-+---------------+---------------+---------------+    +---------+
-|          cTLS Record Layer / Application      |
-+---------------+---------------+---------------+
-~~~~~
+Each specialization produces a new protocol that preserves the security guarantees
+of TLS, but has a unique handshake transcript.  This avoids any need to
+reconstruct a "classic" TLS handshake, but it does not support implementation
+as a compression layer external to the TLS library.
```

Comment:
I think we could remove this "compression" statement. It will  be confusing to future readers.

Reply:
OK, removed.